operators/src/processing/raster_vector_join/aggregator.rs

```diff
@@ -453,4 +453,17 @@ mod tests {
 
         assert_eq!(aggregator.nulls(), &[true, true]);
     }
+
+    #[test]
+    fn value_then_null() {
+        let mut aggregator = FirstValueIntAggregator::new(1).into_typed();
+
+        aggregator.add_value(0, 1337, 1);
+        aggregator.add_null(0);
```

Comment:
I guess the tests need to call more than potentially the number of pristine/null values.
I think this code is still buggy.

And I think the mean variant is too. It all doesn't look that sound :thinking:

Reply:
I fixed the bug where a null value that came after a valid value crashed the server. I fixed that and this tests checks for exactly that.

If there are other bugs, we still have to fin them :D